content/Foundation/Summit/2023/home.pml

```diff
@@ -31,8 +31,62 @@ Sept/Oct 2023, dates TBC
       Latest News
     </h3>
   </div>
+  <h3 class="highlight">
+   Call For Presentations for the seL4 Summit 2023
+  </h3>
+  <ul>
+  <li>Share your seL4 work</li>
+  <li>Share your seL4 experience</li>
+  <li>Share your seL4 thoughts</li>
+  </ul>
+  <p>
+    Check the full <a href="cfp">Call For Presentations</a>. To propose a talk, upload an abstract of one page or less by Friday 31 March 2023 to the submission portal (opening soon).
+  </p>
+</div>
+
+
+<div class="summit_div">
+  <h3 id="summit-about">
+    About
+  </h3>
+  <p>
+    The seL4 Summit is the annual international summit on the <a href="../../../About">seL4 microkernel</a>, the world's most highly assured OS kernel, as well as on all seL4-related technology, tools, infrastructure, products, projects, and people.
+  </p>
+  <p>
+    It aims to gather all the seL4 community to learn, share, and connect:
+  </p>
+    <ul>
+      <li>learn about the seL4 technology, its latest progress, use, successes, challenges, plans;</li>
+      <li>share exciting seL4 development, research, experience, application in the real world;</li>
+      <li>connect with other seL4 developers, users, providers, customers, supporters, potential partners and enthusiasts.</li>
```

Comment:
This is the updated version (I added enthusiasts). Should be replicated in the CfP page